src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  color: $secondary-text-color;
```

Comment:
The variable `$secondary-text-color` is used but not defined in this file. Ensure that it is defined or imported from another SCSS file to avoid compilation errors.